### DISCUSSION TITLE
Proportions of stable / cutting edge / future features

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#121] [State of HTML Design]
In general, the feature lists of _State of X_ surveys skew towards more recent / cutting edge stuff that *does* have some implementations.

Since this is a new survey, there may be value in asking about older, stable features as well, to see whether developers utilize them. Features such as:
- [ARIA](https://github.com/Devographics/surveys/discussions/114)
- [`tabindex`](https://github.com/Devographics/surveys/discussions/113)
- Sectioning elements (`<main>`, `<section>`, `<article>`, `<nav>`, `<footer>` etc)
- [Constraint validation](https://github.com/Devographics/surveys/discussions/120)
- etc

There is also the opposite question: What upcoming features with no implementations and no real spec should we ask about? E.g. would something like [`<breadcrumb>`](https://github.com/Devographics/surveys/discussions/116) be fair game, or is it too early?

**Comment by mfreed7** [comment]
I think it's ok to ask about existing, stable features, but I wouldn't dedicate too much of the survey to that. The interesting question (to me at least) would be what parts about existing features are broken or non-interoperable. That's actionable information. Asking whether folks use existing features doesn't seem too actionable to me, unless you find out *why* they don't use them.
